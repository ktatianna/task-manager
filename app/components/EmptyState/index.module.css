.container {
  display: flex;
  flex-direction: column;
  align-items: center;
  width: 400px;
  margin: 0 auto;
}

.message {
  color: #6b7280;
  font-size: 0.875rem;
  margin-top: 1rem;
  text-align: center;
  width: 100%;
}
