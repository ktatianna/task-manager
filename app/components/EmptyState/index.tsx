import Image from "next/image";
import styles from "./index.module.css";

const EmptyState = () => {
  return (
    <div className={styles.container}>
      <Image src="/lazy.jpg" alt="empty state" width={300} height={180} />
      <span className={styles.message}>You have no tasks yet</span>
    </div>
  );
};

export default EmptyState;