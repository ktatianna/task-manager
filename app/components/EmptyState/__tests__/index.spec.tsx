import { render } from "@testing-library/react";
import EmptyState from "../index";

describe("EmptyState", () => {
  it("should match the snapshot", () => {
    const { container } = render(<EmptyState />);
    expect(container).toMatchSnapshot();
  });
});
