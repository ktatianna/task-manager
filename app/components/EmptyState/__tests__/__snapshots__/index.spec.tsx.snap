// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyState should match the snapshot 1`] = `
<div>
  <div
    class="container"
  >
    <img
      alt="empty state"
      data-nimg="1"
      decoding="async"
      height="180"
      loading="lazy"
      src="/_next/image?url=%2Flazy.jpg&w=640&q=75"
      srcset="/_next/image?url=%2Flazy.jpg&w=384&q=75 1x, /_next/image?url=%2Flazy.jpg&w=640&q=75 2x"
      style="color: transparent;"
      width="300"
    />
    <span
      class="message"
    >
      You have no tasks yet
    </span>
  </div>
</div>
`;
